User/j andrew/whatsnew archive (#2419)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

> ---

Reviewer: in the hub page file
edge-developer\microsoft-edge\index.yml
change from:
```
            - text: What's new in Microsoft Edge DevTools
```
to:
```
            - text: What's New in Microsoft Edge DevTools
```

Author: fixed

Author: fixed

> ---

Reviewer: in the topmost landing page file
edge-developer\microsoft-edge\developer\index.yml
change from:
```
          - text: What's new in Microsoft Edge DevTools
```
to:
```
          - text: What's New in Microsoft Edge DevTools
```

Author: fixed

> ---

Reviewer: in DevTools landing page
edge-developer\microsoft-edge\devtools-guide-chromium\landing\index.yml
change from:
```
          - text: What's new in Microsoft Edge DevTools 
```
to:
```
          - text: What's New in Microsoft Edge DevTools 
```

Author: fixed

> title: What's new in Microsoft Edge DevTools Archives

Reviewer: ```suggestion
title: Archived What's New entries
```

> description: New features and fixes in Microsoft Edge DevTools Archives

Reviewer: ```suggestion
description: New features and fixes in older releases of Microsoft Edge DevTools.
```

author: MSEdgeTeam
ms.author: msedgedevrel
ms.topic: conceptual
ms.prod: microsoft-edge
ms.date: 01/23/2023
---

> # Archived What's New entries

Reviewer: ```suggestion
# What's New archive
```
possibly change to efficient short form, avoids having many words and then finally 'archive'; only 2 words and then 'archive'.
Capital N for best parsing.

Alternative titles:
- What's new archive
- What's New archive
- What's new in Microsoft Edge DevTools Archives
- Archived What's New entries

> This page is an archive for all of the legacy [What's New in DevTools](whats-new.md) entries published after each release.

Reviewer: ```suggestion
The following What's New entries cover older releases.
```
Prefer describing the technology instead of describing the docs.
Draft wording alternatives:
The following features were introduced in earlier releases.

_____________
A separate comment:
Prefer link pattern using one of these options:

For more info see [link]

See also:
* [link]

h2 See also
* [link]

> <!-- adjust placement and indent in toc.yml too -->

Reviewer: ```suggestion
<!-- maintenance notes: adjust placement and indent in toc.yml too -->
```

* [What's New in DevTools (Microsoft Edge 98)](2022/02/devtools.md)
* [What's New in DevTools (Microsoft Edge 97)](2022/01/devtools.md)
* [What's New in DevTools (Microsoft Edge 96)](2021/11/devtools.md)
* [What's New in DevTools (Microsoft Edge 95)](2021/10/devtools.md)
* [What's New in DevTools (Microsoft Edge 94)](2021/09/devtools.md)
* [What's New in DevTools (Microsoft Edge 93)](2021/07/devtools.md)
* [What's New in DevTools (Microsoft Edge 92)](2021/05/devtools.md)
* [What's New in DevTools (Microsoft Edge 91)](2021/04/devtools.md)
* [What's New in DevTools (Microsoft Edge 90)](2021/02/devtools.md)
* [What's New in DevTools (Microsoft Edge 89)](2021/01/devtools.md)
* [What's New in DevTools (Microsoft Edge 88)](2020/11/devtools.md)
* [What's New in DevTools (Microsoft Edge 87)](2020/10/devtools.md)
* [What's New in DevTools (Microsoft Edge 86)](2020/08/devtools.md)
* [What's New in DevTools (Microsoft Edge 85)](2020/06/devtools.md)
* [What's New in DevTools (Microsoft Edge 84)](2020/05/devtools.md)
* [What's New in DevTools (Microsoft Edge 83)](2020/03/devtools.md)
* [What's New in DevTools (Microsoft Edge 81)](2020/01/devtools.md)

> * [What's New in DevTools (Microsoft Edge 80)](2019/12/devtools.md)

Reviewer: un-link the intro section and use instead one of the other standard format options:

Prefer link pattern using one of these options:

For more information see [link]
(this would be in paragraph 1 at top of page)

See also:
* [link]
(this would be below the list of links)

h2 See also 
* [link]

(this would be below the list of links)

> * [What's New in DevTools (Microsoft Edge 80)](2019/12/devtools.md)

Reviewer: ```suggestion
* [What's New in DevTools (Microsoft Edge 80)](2019/12/devtools.md)

See also:
* [What's New in DevTools](whats-new.md)
```

---

> title: Archived What's New entries

Reviewer: ```suggestion
title: What's New in DevTools archive
```

> ---

Reviewer: need entry in toc.yml, need indent old whatsnews in toc:
```
            - name: Microsoft Edge 101
              href: devtools-guide-chromium/whats-new/2022/04/devtools-101.md

            - name: Microsoft Edge 100
              href: devtools-guide-chromium/whats-new/2022/03/devtools-100.md

            - name: What's new archive
              items:
                - name: What's new archive
                  href: devtools-guide-chromium/whats-new/whats-new-archive.md
                  displayName: What's new in Microsoft Edge DevTools Archives # full title

                - name: Microsoft Edge 99
                  href: devtools-guide-chromium/whats-new/2022/03/devtools.md

                - name: Microsoft Edge 98
                  href: devtools-guide-chromium/whats-new/2022/02/devtools.md

                - name: Microsoft Edge 97
                  href: devtools-guide-chromium/whats-new/2022/01/devtools.md

                - name: Microsoft Edge 96
                  href: devtools-guide-chromium/whats-new/2021/11/devtools.md

                - name: Microsoft Edge 95
                  href: devtools-guide-chromium/whats-new/2021/10/devtools.md

                - name: Microsoft Edge 94
                  href: devtools-guide-chromium/whats-new/2021/09/devtools.md

                - name: Microsoft Edge 93
                  href: devtools-guide-chromium/whats-new/2021/07/devtools.md

                - name: Microsoft Edge 92
                  href: devtools-guide-chromium/whats-new/2021/05/devtools.md

                - name: Microsoft Edge 91
                  href: devtools-guide-chromium/whats-new/2021/04/devtools.md

                - name: Microsoft Edge 90
                  href: devtools-guide-chromium/whats-new/2021/02/devtools.md

                - name: Microsoft Edge 89
                  href: devtools-guide-chromium/whats-new/2021/01/devtools.md

                - name: Microsoft Edge 88
                  href: devtools-guide-chromium/whats-new/2020/11/devtools.md

                - name: Microsoft Edge 87
                  href: devtools-guide-chromium/whats-new/2020/10/devtools.md

                - name: Microsoft Edge 86
                  href: devtools-guide-chromium/whats-new/2020/08/devtools.md

                - name: Microsoft Edge 85
                  href: devtools-guide-chromium/whats-new/2020/06/devtools.md

                - name: Microsoft Edge 84
                  href: devtools-guide-chromium/whats-new/2020/05/devtools.md

                - name: Microsoft Edge 83
                  href: devtools-guide-chromium/whats-new/2020/03/devtools.md

                - name: Microsoft Edge 81
                  href: devtools-guide-chromium/whats-new/2020/01/devtools.md

                - name: Microsoft Edge 80
                  href: devtools-guide-chromium/whats-new/2019/12/devtools.md
```

Author: added to the toc.yml

Author: I accepted your suggestions

title: What's new in Microsoft Edge DevTools Archives
description: New features and fixes in Microsoft Edge DevTools Archives
author: MSEdgeTeam
ms.author: msedgedevrel
ms.topic: conceptual
ms.prod: microsoft-edge
ms.date: 01/23/2023
---

> # What's new in Microsoft Edge DevTools Archives

Reviewer: ```suggestion
# Archived What's New entries
```
Trying to put leading word = key word on left.

'new' is capitalized in some spots in current design:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/01/devtools-109

If retitle this article, do in all usual spots:
* title: yaml
* h1
* toc entry ("name:" field, or "displayName:" field when need to support lookup of verbose full title)
* link text in any other files that point to here:   existing What's New page.

Author: fixed

> This page is an archive for all of the legacy [What's New in DevTools](whats-new.md) published after each release. 

Reviewer: ```suggestion
This page is an archive for all of the legacy [What's New in DevTools](whats-new.md) entries published after each release. 
```
or "pages"

> <!-- add new pages to toc too -->

Reviewer: ```suggestion
<!-- adjust placement and indent in toc.yml too -->
```
maintenance notes, proved to be needed

> * [What's New in DevTools (Microsoft Edge 109)](2023/01/devtools-109.md)
> * [What's New in DevTools (Microsoft Edge 108)](2022/12/devtools-108.md)
> * [What's New in DevTools (Microsoft Edge 107)](2022/10/devtools-107.md)
> * [What's New in DevTools (Microsoft Edge 106)](2022/09/devtools-106.md)
> * [What's New in DevTools (Microsoft Edge 105)](2022/09/devtools-105.md)
> * [What's New in DevTools (Microsoft Edge 104)](2022/08/devtools-104.md)
> * [What's New in DevTools (Microsoft Edge 103)](2022/06/devtools-103.md)
> * [What's New in DevTools (Microsoft Edge 102)](2022/05/devtools-102.md)
> * [What's New in DevTools (Microsoft Edge 101)](2022/04/devtools-101.md)
> * [What's New in DevTools (Microsoft Edge 100)](2022/03/devtools-100.md)

Reviewer: ```suggestion
```
if i understand the intended redesign
